<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>932</width>
    <height>746</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QComboBox" name="selector_action">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>340</y>
      <width>651</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>130</y>
      <width>82</width>
      <height>120</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QPushButton" name="set_X">
       <property name="text">
        <string>SET X</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="set_Y">
       <property name="text">
        <string>SET Y</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="set_Z">
       <property name="text">
        <string>SET Z</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>380</y>
      <width>871</width>
      <height>299</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="100,0,100">
     <item>
      <widget class="Widget_tableWithSearch" name="items_to_select"/>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="rightMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QPushButton" name="selector_to">
         <property name="maximumSize">
          <size>
           <width>31</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>-&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="selector_cancel">
         <property name="maximumSize">
          <size>
           <width>31</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>&lt;-</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Widget_tableWithSearch" name="items_selected"/>
     </item>
    </layout>
   </widget>
   <widget class="Widget_listWithSearch" name="available_plot_data">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>50</y>
      <width>651</width>
      <height>251</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="reset_colors">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>310</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="maximumSize">
     <size>
      <width>150</width>
      <height>150</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">color: red</string>
    </property>
    <property name="text">
     <string>RESET</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Select X-Y-Z data to plot</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>310</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Filter attributes</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>932</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Widget_listWithSearch</class>
   <extends>QWidget</extends>
   <header>optimeed.visualize.widgets.widget_listWithSearch</header>
  </customwidget>
  <customwidget>
   <class>Widget_tableWithSearch</class>
   <extends>QWidget</extends>
   <header>optimeed.visualize.widgets.widget_tableWithSearch</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>selector_action</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
